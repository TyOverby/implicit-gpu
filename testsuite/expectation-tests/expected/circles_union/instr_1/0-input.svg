version https://git-lfs.github.com/spec/v1
oid sha256:a3949ca6b5e6d83d0ac9186090773344e50c153227eee7f79cd444b4dc4c9917
size 17417
