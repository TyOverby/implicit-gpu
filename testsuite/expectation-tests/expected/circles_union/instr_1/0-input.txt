124 line segments

Line Segment 0 
2 points
Clockwise? false
(3.0,13.0)
(3.0,13.0)

Line Segment 1 
2 points
Clockwise? false
(3.0,13.0)
(3.0,13.0)

Line Segment 2 
2 points
Clockwise? true
(3.0,13.0)
(3.0501516,12.0)

Line Segment 3 
2 points
Clockwise? true
(3.0501516,12.0)
(3.2023916,11.0)

Line Segment 4 
2 points
Clockwise? false
(3.0501516,14.0)
(3.0,13.0)

Line Segment 5 
2 points
Clockwise? true
(3.2023916,11.0)
(3.4617925,10.0)

Line Segment 6 
2 points
Clockwise? false
(3.2023916,15.0)
(3.0501516,14.0)

Line Segment 7 
2 points
Clockwise? true
(3.4617925,10.0)
(3.8359776,9.0)

Line Segment 8 
2 points
Clockwise? false
(3.4617925,16.0)
(3.2023916,15.0)

Line Segment 9 
2 points
Clockwise? true
(3.8359776,9.0)
(4.0,8.661701)

Line Segment 10 
2 points
Clockwise? false
(3.8359776,17.0)
(3.4617925,16.0)

Line Segment 11 
2 points
Clockwise? true
(4.0,8.661701)
(4.3430977,8.0)

Line Segment 12 
2 points
Clockwise? false
(4.0,17.338299)
(3.8359776,17.0)

Line Segment 13 
2 points
Clockwise? true
(4.3430977,8.0)
(5.0,7.0)

Line Segment 14 
2 points
Clockwise? false
(4.3430977,18.0)
(4.0,17.338299)

Line Segment 15 
2 points
Clockwise? false
(5.0,7.0)
(5.0,7.0)

Line Segment 16 
2 points
Clockwise? true
(5.0,7.0)
(5.862445,6.0)

Line Segment 17 
2 points
Clockwise? false
(5.0,19.0)
(4.3430977,18.0)

Line Segment 18 
2 points
Clockwise? false
(5.0,19.0)
(5.0,19.0)

Line Segment 19 
2 points
Clockwise? true
(5.862445,6.0)
(6.0,5.862445)

Line Segment 20 
2 points
Clockwise? false
(5.862445,20.0)
(5.0,19.0)

Line Segment 21 
2 points
Clockwise? true
(6.0,5.862445)
(7.0,5.0)

Line Segment 22 
2 points
Clockwise? false
(6.0,20.137556)
(5.862445,20.0)

Line Segment 23 
2 points
Clockwise? false
(7.0,5.0)
(7.0,5.0)

Line Segment 24 
2 points
Clockwise? true
(7.0,5.0)
(8.0,4.3430977)

Line Segment 25 
2 points
Clockwise? false
(7.0,21.0)
(6.0,20.137556)

Line Segment 26 
2 points
Clockwise? false
(7.0,21.0)
(7.0,21.0)

Line Segment 27 
2 points
Clockwise? true
(8.0,4.3430977)
(8.661701,4.0)

Line Segment 28 
2 points
Clockwise? false
(8.0,21.656902)
(7.0,21.0)

Line Segment 29 
2 points
Clockwise? true
(8.661701,4.0)
(9.0,3.8359776)

Line Segment 30 
2 points
Clockwise? false
(8.661701,22.0)
(8.0,21.656902)

Line Segment 31 
2 points
Clockwise? true
(9.0,3.8359776)
(10.0,3.4617925)

Line Segment 32 
2 points
Clockwise? false
(9.0,22.164022)
(8.661701,22.0)

Line Segment 33 
2 points
Clockwise? true
(10.0,3.4617925)
(11.0,3.2023916)

Line Segment 34 
2 points
Clockwise? false
(10.0,22.538208)
(9.0,22.164022)

Line Segment 35 
2 points
Clockwise? true
(11.0,3.2023916)
(12.0,3.0501516)

Line Segment 36 
2 points
Clockwise? false
(11.0,22.79761)
(10.0,22.538208)

Line Segment 37 
2 points
Clockwise? true
(12.0,3.0501516)
(13.0,3.0)

Line Segment 38 
2 points
Clockwise? false
(12.0,22.949848)
(11.0,22.79761)

Line Segment 39 
2 points
Clockwise? false
(13.0,3.0)
(13.0,3.0)

Line Segment 40 
2 points
Clockwise? false
(13.0,3.0)
(13.0,3.0)

Line Segment 41 
2 points
Clockwise? true
(13.0,3.0)
(14.0,3.0501516)

Line Segment 42 
2 points
Clockwise? false
(13.0,23.0)
(12.0,22.949848)

Line Segment 43 
2 points
Clockwise? false
(13.0,23.0)
(13.0,23.0)

Line Segment 44 
2 points
Clockwise? false
(13.050152,24.0)
(13.0,23.0)

Line Segment 45 
2 points
Clockwise? false
(13.202392,25.0)
(13.050152,24.0)

Line Segment 46 
2 points
Clockwise? false
(13.461792,26.0)
(13.202392,25.0)

Line Segment 47 
2 points
Clockwise? false
(13.835978,27.0)
(13.461792,26.0)

Line Segment 48 
2 points
Clockwise? true
(14.0,3.0501516)
(15.0,3.2023916)

Line Segment 49 
2 points
Clockwise? false
(14.0,27.338299)
(13.835978,27.0)

Line Segment 50 
2 points
Clockwise? false
(14.343098,28.0)
(14.0,27.338299)

Line Segment 51 
2 points
Clockwise? true
(15.0,3.2023916)
(16.0,3.4617925)

Line Segment 52 
2 points
Clockwise? false
(15.0,29.0)
(14.343098,28.0)

Line Segment 53 
2 points
Clockwise? false
(15.0,29.0)
(15.0,29.0)

Line Segment 54 
2 points
Clockwise? false
(15.862445,30.0)
(15.0,29.0)

Line Segment 55 
2 points
Clockwise? true
(16.0,3.4617925)
(17.0,3.8359776)

Line Segment 56 
2 points
Clockwise? false
(16.0,30.137556)
(15.862445,30.0)

Line Segment 57 
2 points
Clockwise? true
(17.0,3.8359776)
(17.338299,4.0)

Line Segment 58 
2 points
Clockwise? false
(17.0,31.0)
(16.0,30.137556)

Line Segment 59 
2 points
Clockwise? false
(17.0,31.0)
(17.0,31.0)

Line Segment 60 
2 points
Clockwise? true
(17.338299,4.0)
(18.0,4.3430977)

Line Segment 61 
2 points
Clockwise? true
(18.0,4.3430977)
(19.0,5.0)

Line Segment 62 
2 points
Clockwise? false
(18.0,31.656902)
(17.0,31.0)

Line Segment 63 
2 points
Clockwise? false
(18.661701,32.0)
(18.0,31.656902)

Line Segment 64 
2 points
Clockwise? false
(19.0,5.0)
(19.0,5.0)

Line Segment 65 
2 points
Clockwise? true
(19.0,5.0)
(20.0,5.862445)

Line Segment 66 
2 points
Clockwise? false
(19.0,32.164024)
(18.661701,32.0)

Line Segment 67 
2 points
Clockwise? true
(20.0,5.862445)
(20.137556,6.0)

Line Segment 68 
2 points
Clockwise? false
(20.0,32.538208)
(19.0,32.164024)

Line Segment 69 
2 points
Clockwise? true
(20.137556,6.0)
(21.0,7.0)

Line Segment 70 
2 points
Clockwise? false
(21.0,7.0)
(21.0,7.0)

Line Segment 71 
2 points
Clockwise? true
(21.0,7.0)
(21.656902,8.0)

Line Segment 72 
2 points
Clockwise? false
(21.0,32.797607)
(20.0,32.538208)

Line Segment 73 
2 points
Clockwise? true
(21.656902,8.0)
(22.0,8.661701)

Line Segment 74 
2 points
Clockwise? true
(22.0,8.661701)
(22.164022,9.0)

Line Segment 75 
2 points
Clockwise? false
(22.0,32.94985)
(21.0,32.797607)

Line Segment 76 
2 points
Clockwise? true
(22.164022,9.0)
(22.538208,10.0)

Line Segment 77 
2 points
Clockwise? true
(22.538208,10.0)
(22.79761,11.0)

Line Segment 78 
2 points
Clockwise? true
(22.79761,11.0)
(22.949848,12.0)

Line Segment 79 
2 points
Clockwise? true
(22.949848,12.0)
(23.0,13.0)

Line Segment 80 
2 points
Clockwise? false
(23.0,13.0)
(23.0,13.0)

Line Segment 81 
2 points
Clockwise? true
(23.0,13.0)
(24.0,13.050152)

Line Segment 82 
2 points
Clockwise? false
(23.0,33.0)
(22.0,32.94985)

Line Segment 83 
2 points
Clockwise? false
(23.0,33.0)
(23.0,33.0)

Line Segment 84 
2 points
Clockwise? false
(23.0,33.0)
(23.0,33.0)

Line Segment 85 
2 points
Clockwise? true
(24.0,13.050152)
(25.0,13.202392)

Line Segment 86 
2 points
Clockwise? false
(24.0,32.94985)
(23.0,33.0)

Line Segment 87 
2 points
Clockwise? true
(25.0,13.202392)
(26.0,13.461792)

Line Segment 88 
2 points
Clockwise? false
(25.0,32.797607)
(24.0,32.94985)

Line Segment 89 
2 points
Clockwise? true
(26.0,13.461792)
(27.0,13.835978)

Line Segment 90 
2 points
Clockwise? false
(26.0,32.538208)
(25.0,32.797607)

Line Segment 91 
2 points
Clockwise? true
(27.0,13.835978)
(27.338299,14.0)

Line Segment 92 
2 points
Clockwise? false
(27.0,32.164024)
(26.0,32.538208)

Line Segment 93 
2 points
Clockwise? true
(27.338299,14.0)
(28.0,14.343098)

Line Segment 94 
2 points
Clockwise? false
(27.338299,32.0)
(27.0,32.164024)

Line Segment 95 
2 points
Clockwise? true
(28.0,14.343098)
(29.0,15.0)

Line Segment 96 
2 points
Clockwise? false
(28.0,31.656902)
(27.338299,32.0)

Line Segment 97 
2 points
Clockwise? false
(29.0,15.0)
(29.0,15.0)

Line Segment 98 
2 points
Clockwise? true
(29.0,15.0)
(30.0,15.862445)

Line Segment 99 
2 points
Clockwise? false
(29.0,31.0)
(28.0,31.656902)

Line Segment 100 
2 points
Clockwise? false
(29.0,31.0)
(29.0,31.0)

Line Segment 101 
2 points
Clockwise? true
(30.0,15.862445)
(30.137556,16.0)

Line Segment 102 
2 points
Clockwise? false
(30.0,30.137556)
(29.0,31.0)

Line Segment 103 
2 points
Clockwise? true
(30.137556,16.0)
(31.0,17.0)

Line Segment 104 
2 points
Clockwise? false
(30.137556,30.0)
(30.0,30.137556)

Line Segment 105 
2 points
Clockwise? false
(31.0,17.0)
(31.0,17.0)

Line Segment 106 
2 points
Clockwise? true
(31.0,17.0)
(31.656902,18.0)

Line Segment 107 
2 points
Clockwise? false
(31.0,29.0)
(30.137556,30.0)

Line Segment 108 
2 points
Clockwise? false
(31.0,29.0)
(31.0,29.0)

Line Segment 109 
2 points
Clockwise? true
(31.656902,18.0)
(32.0,18.661701)

Line Segment 110 
2 points
Clockwise? false
(31.656902,28.0)
(31.0,29.0)

Line Segment 111 
2 points
Clockwise? true
(32.0,18.661701)
(32.164024,19.0)

Line Segment 112 
2 points
Clockwise? false
(32.0,27.338299)
(31.656902,28.0)

Line Segment 113 
2 points
Clockwise? true
(32.164024,19.0)
(32.538208,20.0)

Line Segment 114 
2 points
Clockwise? false
(32.164024,27.0)
(32.0,27.338299)

Line Segment 115 
2 points
Clockwise? true
(32.538208,20.0)
(32.797607,21.0)

Line Segment 116 
2 points
Clockwise? false
(32.538208,26.0)
(32.164024,27.0)

Line Segment 117 
2 points
Clockwise? true
(32.797607,21.0)
(32.94985,22.0)

Line Segment 118 
2 points
Clockwise? false
(32.797607,25.0)
(32.538208,26.0)

Line Segment 119 
2 points
Clockwise? true
(32.94985,22.0)
(33.0,23.0)

Line Segment 120 
2 points
Clockwise? false
(32.94985,24.0)
(32.797607,25.0)

Line Segment 121 
2 points
Clockwise? false
(33.0,23.0)
(32.94985,24.0)

Line Segment 122 
2 points
Clockwise? false
(33.0,23.0)
(33.0,23.0)

Line Segment 123 
2 points
Clockwise? false
(33.0,23.0)
(33.0,23.0)
